APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -436,6 +440,24 @@ - (UITableViewCell *) tableView:(UITableView *)tableView cellForRowAtIndexPath:(
     return cell;
 }
 
+- (void) setEditing:(BOOL) isEditing
+{
+    [super setEditing:isEditing];
+    
+    NSArray* visibleCells = [self.tableView visibleCells];
+    for (UITableViewCell* cell in visibleCells) {
```

Comment:
A simple reload of the table data should work, let me try.  My original goal was trying to minimize the potential impact of my code changes.